src/CMakeLists.txt

```diff
@@ -1,22 +1,34 @@
 # Filling templates with shaders code.
 file(READ simple.vert SIMPLE_VERT)
 file(READ simple.frag SIMPLE_FRAG)
-configure_file(shaders.cpp.in shaders.cpp)
-configure_file(shaders.hpp ${CMAKE_CURRENT_BINARY_DIR})
+configure_file(shaders.cpp.in shaders.cpp @ONLY)
```

Comment:
Always try to use `@ONLY` mode (and `@PLACEHOLSERS@` like this). Leave the `${var}` to Shell scripts :)